### PR TITLE
<InlineText />Removed Tab Index so it focuses the input when I click on the pencil (Safari and FireFox)

### PR DESCRIPTION
**Pull request by yoyis3000**
### Summary
On Safari and Firefox, clicking on the pencil icon would just cause the pencil to disappear and it wont focus the input for InlineText. Removing the tab index allowed the container to be focused when the pencil icon is clicked. 

#### Before
inline text before

#### After
inline text after

### Testing
Click on Pencil icon or container and make sure it gets focused.

### CODE REVIEW

File: table/src/InlineText.jsx [hunk 1]
```diff
@@ -51,6 +51,8 @@ export default class InlineText extends React.Component {
   onBlur = () => {
     this.props.onBlur();
     this.onSubmit();
+    delete this.container.removeAttribute('tabIndex');
+    console.log('look at me blur');
```
Comment (dancingshell) [inline table/src/InlineText.jsx]: remove log

File: table/src/InlineText.jsx [hunk 2]
```diff
@@ -51,6 +51,8 @@ export default class InlineText extends React.Component {
   onBlur = () => {
     this.props.onBlur();
     this.onSubmit();
+    delete this.container.removeAttribute('tabIndex');
```
Comment (dancingshell) [inline table/src/InlineText.jsx]: if you already removed the `tabIndex` from the element, fo you have to remove it again here?

also the `delete` I'm not sure the delete is doing anything here

Comment (yoyis3000) [reply table/src/InlineText.jsx]: I forgot to _delete_ this.

File: table/src/InlineText.jsx (hunk 1 above)
Comment (yoyis3000) [reply table/src/InlineText.jsx]: will do!

**Pull request merged**